scout/server/blueprints/cases/templates/cases/case_bionano.html

```diff
@@ -107,6 +107,8 @@ <h4 class="mt-3">Case: {{case.display_name}}</h4>
                     {% endif %}
                     {% if ind.confirmed_sex %}
                       <i class="fa fa-check"></i>
+                    {% else %}
+                      <span class="fa fa-exclamation-circle text-danger" data-bs-toggle='tooltip' title="Sex is not confirmed."></span>
```

Comment:
Looks good, should work just fine! Feel free to just be content with this, but if you want, I'm sure you also noticed those were rather duplicated looking parts you found. We in theory work by the three strikes model: first time, just do it, second time, copy and smile, then third time seriously consider refactoring a tad. In this case it's so small, it doesn't really matter either way. And you could argue there are just two that are perfectly identical. But if you like, add a small macro to reuse to show the sex table cell div. It can remain unique for the case report, and no need to worry about including the interactive version select in the macro.

Reply:
Yes, I was indeed thinking about this. Sounds like good learning to pull this out into a macro. I'll give it a go